packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-env browser */
+
+/* global Webex */
+
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+/* eslint-disable no-global-assign */
+/* eslint-disable no-multi-assign */
+
+// Globals
+let webex;
+
+// Collapsible Setup
+const collapsibles = document.querySelectorAll('.collapsible');
+
+function collapse() {
+  const prevState = this.getAttribute('data-collapsed');
+  const newState = prevState === 'true' ? 'false' : 'true';
+
+  this.setAttribute('data-collapsed', newState);
+  this.nextElementSibling.setAttribute('data-collapsed', newState);
+}
+
+collapsibles.forEach(
+  (collapsible) => {
+    collapsible.addEventListener('click', collapse);
+  }
+);
+
+// Authentication Section
```

Comment:
I like longer comment delineators, I find it makes it easier to glance over sections
```suggestion
// Authentication Section --------------------------------------------------
```

Reply:
Implemented.